### ISSUE TITLE
display-menu truncates title

### ISSUE THREAD

**Issue by bchallenor** [tmux/tmux#2231]
### Issue description

`display-menu` does not take into account the title when calculating the width of the menu. If the title is longer than the menu items, it is truncated.

For example, with the following menu, the title is displayed as "This is a re":
```
display-menu -T "This is a really long title" -- "Tiny row" t 'display-message "Hi"'
```

### Required information

```
$ uname -sp && ./tmux -V && echo $TERM
Linux unknown
tmux next-3.2
screen
```

**Comment by nicm** [comment]
This should be fixed now by 26e8e467e8e4d3e4f4596b9ce50832735cdc4d50. Let me know if you still see problems, thanks!

**Comment by bchallenor** [comment]
Can confirm this fixes the issue. Thanks!
